Resources/config/serializer/PhpcrOdmResource.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<serializer>
+    <class name="Symfony\Cmf\Component\Resource\Repository\Resource\PhpcrOdmResource" xmlns:h="https://github.com/willdurand/Hateoas">
+        <h:relation rel="self">
+            <h:href route="_cmf_resource">
+                <h:parameter name="repositoryName" value="testrepo" />
+                <h:parameter name="path" value="expr(object.getPath())" />
+            </h:href>
+        </h:relation>
+        <virtual-property serialized-name="children" max-depth="0" method="listChildren">
+            <xml-list inline="true" entry-name="foobar" />
+        </virtual-property>
```

Comment:
Fixed. "resources" was the key used in the `ResourceCollection ` iterator. Have had to implement a custom "handler" to normalize it. Not sure if there is a better way.